swift/Colorspace.swift

```diff
@@ -0,0 +1,21 @@
+import thorvg
+
+/// Specifiies the methods of combining the 8-bit color channels into 32-bit color.
+public enum Colorspace {
+    /// The channels are joined in the order: alpha, blue, green, red. Colors are alpha-premultiplied. (a << 24 | b << 16 | g << 8 | r)
+    case abgr
+    /// The channels are joined in the order: alpha, red, green, blue. Colors are alpha-premultiplied. (a << 24 | r << 16 | g << 8 | b)
+    case argb
+}
```

Comment:
These are not really color spaces but pixel formats. Also, you want to use the term component instead of channel. A channel refers to the set of all the values, for example, saying that an image has an alpha channel means that all the pixels have transparency information associated to them. 

So, when you say:

> The channels are joined in the order: alpha, red, green, blue

That really means that somehow an image is stored in a way that all the alpha values come first, then all the red, and so on.

Reply:
That's a fair call. I've gotten swept up in the ThorVG naming without actually questioning whether it is a good name!

I'll update the naming to move to `PixelFormat`.